examples/bri-3/src/bri/transactions/agents/transactionStorage.agent.ts

```diff
@@ -1,13 +1,36 @@
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
 import { Injectable, NotFoundException } from '@nestjs/common';
 import { PrismaService } from '../../../../prisma/prisma.service';
 import { NOT_FOUND_ERR_MESSAGE } from '..//api/err.messages';
 import { Transaction } from '../models/transaction';
 
 @Injectable()
 export class TransactionStorageAgent extends PrismaService {
+  constructor(@InjectMapper() private autoMapper: Mapper) {
+    super();
+  }
+
+  async getAllTransactions(): Promise<Transaction[]> {
+    const transactionModels = await this.transaction.findMany();
+    return transactionModels.map((transactionModel) => {
+      return new Transaction(
+        transactionModel.id,
+        transactionModel.nonce,
+        transactionModel.workflowInstanceId,
+        transactionModel.workstepInstanceId,
+        null, // TODO: transactionModel.from once BpiAccount in the prisma schema,
```

Comment:
i added these changes in separate branch i think
we can add them once you merge this one if its too much to add here